src/backend/db/products.js

```diff
@@ -19,6 +19,8 @@ export const products = [
     avalQty: 10,
     brand: "Nike",
     categoryName: "men",
+    inCart: false,
+    inWishlist: false,
```

Comment:
I implemented it without adding this specification in backend products, not sure which approach is better

Reply:
Same confusion as @abhij1607.

Reply:
Will look at another solution as well.